### PR TITLE
added paper clip image uploder and latter opener

### PR DESCRIPTION
**Pull request by joney000**
added file / image upload functionality along with email verification and letter opener.

### CODE REVIEW

File: app/views/articles/index.html.haml [hunk 1]
```diff
@@ -23,8 +23,10 @@
...
         %td= article.text
+        %td= image_tag article.image.url(:thumb)
         %td= link_to 'Show', article_path(article)
         %td= link_to 'Edit', edit_article_path(article)
         %td= link_to 'Destroy', article_path(article), |
             method: :delete,                          |
-            data: { confirm: 'Are you sure?' }        |
\ No newline at end of file
+            data: { confirm: 'Are you sure?' }        |
+    = will_paginate @articles
```
Comment (coolRaider) [inline app/views/articles/index.html.haml]: No newline at end of file. Please check the other files too.

Comment (coolRaider) [inline app/views/articles/index.html.haml]: Why so many images for the logo?

Comment (joney000) [inline app/views/articles/index.html.haml]: each image for one article.

File: app/views/articles/show.html.haml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+= image_tag@article.image.url(:medium)
```
Comment (alienware) [inline app/views/articles/show.html.haml:1]: Give some nice bootstrap classes to images also. Go go.

File: config/environments/development.rb [hunk 3]
```diff
@@ -23,7 +23,7 @@
     :port                 => 587,  
     :domain               => "gmail.com",  
     :user_name            => "jaswantsinghyadav007@gmail.com",  
-    :password             => ,  
+    :password             => "",
```
Comment (alienware) [inline config/environments/development.rb:26]: Please please, do not put your password here. 👊

File: app/views/articles/index.html.haml (hunk 1 above)
Comment (alienware) [inline app/views/articles/index.html.haml]: Take out public/system out of VC.

Comment (joney000) [inline app/views/articles/index.html.haml]: done

File: config/environments/development.rb (hunk 3 above)
Comment (joney000) [inline config/environments/development.rb:26]: it is set empty .

**Pull request merged**